eoscostarica/eos-rate
# Update start script

### GH Issue

Added `make dev` command for running local development environment. 

Previous command `make start` will be used on the backend server

### Steps to test

1. run `make dev`
2. visit `localhost:3000`
3. you should see the eos rate page
